code/processes/tickerlogreplay.q

```diff
@@ -351,7 +353,7 @@ merge:{[dir;pt;tablename;mergelimits;h]
 
 if[.replay.autoreplay;
   // grab stp meta table before replaying logs if in segmentedmode
-  // TO DO - define meta table here for replay flexibility
+  stpmeta:([]seq:(), logname:(), start:(), end:(), tbls:(), msgcount:(), schema:(), additional:());
```

Comment:
Not sure this should just be an empty table. Is it being used anywhere?
Also I think most of these columns should be typed

Reply:
Also semicolons rather than commas